Rubberduck.Inspections/QuickFixes/RemoveEmptyIfBlockQuickFix.cs

```diff
@@ -36,14 +36,26 @@ private void UpdateContext(VBAParser.IfStmtContext context, IModuleRewriter rewr
             var elseBlock = context.elseBlock();
             var elseIfBlock = context.elseIfBlock().FirstOrDefault();
 
+            if (BlockHasDeclaration(context.block()))
+            {
+                rewriter.InsertBefore(context.Start.TokenIndex, context.block().GetText());
+            }
+
             if (elseIfBlock != null)
             {
-                rewriter.RemoveRange(context.IF().Symbol.TokenIndex, context.endOfStatement().Stop.TokenIndex);
+                rewriter.RemoveRange(context.IF().Symbol.TokenIndex, context.block()?.Stop.TokenIndex ?? context.endOfStatement().Stop.TokenIndex);
```

Comment:
@MDoerner will want to weight in on this one, his upcoming PR is going to affect this.